core/modules/layout/layout.admin.inc

```diff
@@ -271,12 +271,21 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     }
   }
 
+  $items = array(
+    t('Content: &nbsp; <code>node/%</code>'),
+    t('User accounts: &nbsp; <code>user/%</code>'),
+    t('Taxonomy terms: &nbsp; <code>taxonomy/term/%</code>'),
+    t('Comments: &nbsp; <code>comment/%</code>'),
```

Comment:
These `&nbsp;`s are not required here (the less HTML we show to translators, the better). If there needs to be spacing between colon and the code tag (which I don't think it does), then let's add it via CSS.

Reply:
It was really hard to read without the extra space. We could add it via CSS, but on this page only? Or for all code tags in Seven?

Reply:
I think that this would be a general issue, so should be fixed in Seven. I agree that this can be done in a follow-up 👍

Reply:
fixed in latest pr